docs/markdown/Getting Help/the-pants-community/team.md

```diff
@@ -46,6 +46,8 @@ Pants open source project has had many team members in over a decade. The curren
 | **NAME**                | **PROUDEST CONTRIBUTION**                                                                                |
 |-------------------------|----------------------------------------------------------------------------------------------------------|
 | **Chris Williams**      |                                                                                                          |
+| **Daniel Goldman**      | "Pants metaprogramming"                                                                                                         |
+| **Darcy Shen**      |                                                                                                          |
```

Comment:
Exploring the best practice for Data and ML Engineering in Python and Scala

Reply:
That's my contribution as a full time Data Engineer who are using Pants and giving feedbacks.

Reply:
Done!